cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -24,6 +24,10 @@ private Element getRealParent(Expr expr) {
   result = expr.getParentWithConversions()
   or
   result.(Destructor).getADestruction() = expr
+  or
+  result.(Expr).getASyntheticDestructor() = expr
+  or
+  result.(Stmt).getASyntheticDestructor() = expr
```

Comment:
Is it ever the case that an `expr` has multiple parents with these new cases included?